nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AttributesToJSON.java

```diff
@@ -212,10 +212,7 @@ protected void init(final ProcessorInitializationContext context) {
             if (ats != null) {
                 Set<String> result = new HashSet<>(ats.length);
                 for (String str : ats) {
-                    String trim = str.trim();
-                    if (!atrsToExclude.contains(trim)) {
```

Comment:
With this change, the `atrsToExclude` parameter no longer appears to be used, so the method signature should be adjusted and calling code adjusted.

Reply:
Thanks for the review! I updated the signature in ff81c1c810bb972ebe7de9d9915fc024a76ec0f2